hs_core/templates/pages/genericresource.html

```diff
@@ -811,21 +817,17 @@ <h3>Resource Specific</h3>
 
         {# ======= Comments ======= #}
         {% include "resource-landing-page/comments.html" %}
-
-        {%  block modal %}
-            {% include "resource-landing-page/modals.html" %}
-            {% include "includes/author-preview.html" %}
-            {% include "resource-landing-page/coordinates_picker.html" %}
-        {%  endblock %}
         </div>
 {% endblock %}
 
 {% block extra_js %}
+    {% set_page_permissions page %}
     {{ block.super }}
     <script>
       {# GLOBAL DECLARATIONS FOR RESOURCE LANDING PAGE SCRIPTS #}
       const RES_KEYWORDS = {{ keywords|safe }};
       const SELF_ACCESS_LEVEL = "{{ self_access_level }}";
```

Comment:
just so I understand here, the `|` was removed, correct? Does this have some significance for our version of Django?

Reply:
Could you elaborate on this?

Reply:
Line 827 is showing pink, as though a character was removed:
![image](https://user-images.githubusercontent.com/45397838/61070635-2f68ad80-a3dd-11e9-9152-94652f96141b.png)